openvdb/openvdb/cmd/CMakeLists.txt

```diff
@@ -81,17 +81,40 @@ endif()
 #### vdb_render
 
 if(OPENVDB_BUILD_VDB_RENDER)
-  find_package(Imath CONFIG)
-  if (NOT TARGET Imath::Imath)
-    find_package(IlmBase ${MINIMUM_ILMBASE_VERSION} REQUIRED COMPONENTS Half Iex IlmThread Imath)
-    find_package(OpenEXR ${MINIMUM_OPENEXR_VERSION} REQUIRED COMPONENTS IlmImf)
-  else()
-    find_package(OpenEXR CONFIG)
+
+  if(USE_PNG)
+    find_package(PNG REQUIRED)
+  endif()
```

Comment:
Kind of annoying you have to manually enable this (i.e. `-DUSE_PNG=ON`). @jmlait I know we discussed auto enabling dependencies if they're found and you were against it. I agree for the core library but perhaps for binary features this would be nice?